### PR TITLE
11.0

### PR DESCRIPTION
**Pull request by saif-serpentcs**
11.0 Module compatible.

### CODE REVIEW

File: jasper_reports/JasperReports/browse_data_generator.py [hunk 1]
```diff
@@ -320,7 +320,7 @@ def generate(self, file_name):
         reportCopies = self.report.copies or 1
         sequence = 0
         copiesField = self.report.copies_field
-        for record in self.env[self.model].browse(self.ids):
+        for record in self.env['res.users'].browse(self.ids):
```
Comment (JayVora-SerpentCS) [inline jasper_reports/JasperReports/browse_data_generator.py:323]: Please do not keep static code.

File: jasper_reports/__manifest__.py [hunk 2]
```diff
@@ -45,13 +45,14 @@
         'images/jasper_reports.png'
     ],
     "depends": [
-        "sale", "jasper_load"
+        "sale", "jasper_load", "report_jasper"
     ],
```
Comment (JayVora-SerpentCS) [inline jasper_reports/__manifest__.py:49]: is report_jasper a module?

File: jasper_reports/JasperReports/browse_data_generator.py (hunk 1 above)
Comment (saif-serpentcs) [reply jasper_reports/JasperReports/browse_data_generator.py:323]: Yes sir, for the testing set the static, I will remove it and make it self.model, after print the report.

File: jasper_reports/__manifest__.py (hunk 2 above)
Comment (saif-serpentcs) [reply jasper_reports/__manifest__.py:49]: Yes, sir.

Comment (JayVora-SerpentCS) [reply jasper_reports/__manifest__.py:49]: I see you added with this.

**Pull request merged**